### DISCUSSION TITLE
Is there a way to escape a set of characters?

### DISCUSSION THREAD

**Discussion by architchandra** [mundschenk-at/php-typography#156] [Ideas]
In general, I want to have the functionality that `php-typography` offers, but in certain situations, I want to use certain patterns literally that would have otherwise been converted. eg. I want to use (c) in some cases without it getting converted to ©.

I couldn't find anything in the documentation that would help with this (usual escaping characters are not working). Maybe I'm missing something obvious.

What should I do?

**Comment by mundschenk-at** [comment]
There is no "escaping" on the character level, but wrapping it in `<span class="noTypo"></span>` will do the trick (or whatever "ignore" class you prefer, it depends on the settings).

**Comment by architchandra** [comment]
Can I then make a request for you to consider adding _escaping_ at the character level in the future?

**Comment by mundschenk-at** [comment]
If you've got a specific proposal, please create a new issue as a feature request. Otherwise, I'll think about it, but it's unlikely to happen in the near future.  I can't think of a non-HTML based version that won't cause a lot of side-effects for existing installations.

**Comment by architchandra** [comment]
Thanks. I'll think of the specific details and follow-up with you.
